src/data/constants/courseInfo.js

```diff
@@ -7,6 +7,9 @@ const CREATE_COURSE_FAIL = 'CREATE_COURSE_FAIL';
 const EDIT_COURSE_INFO = 'EDIT_COURSE_INFO';
 const EDIT_COURSE_SUCCESS = 'EDIT_COURSE_SUCCESS';
 const EDIT_COURSE_FAIL = 'EDIT_COURSE_FAIL';
+export const CREATE_COURSE_RUN = 'CREATE_COURSE_RUN';
+export const CREATE_COURSE_RUN_SUCCESS = 'CREATE_COURSE_RUN_SUCCESS';
+export const CREATE_COURSE_RUN_FAIL = 'CREATE_COURSE_RUN_FAIL';
```

Comment:
Listen. I am in favor of the inline export myself. But I believe some recommendation (some linter?) is pushing us toward the style of a large export statement at the bottom. Get on board for the big win.

Reply:
I don't believe the linter cares and that it is mostly personal preference. It was referenced to me originally by Stank and reinforced by Alasdair. I also have some preference towards it since it is typically how we export all of our other files and components. Alasdair did say that he preferred it especially for files with many exports, but when there were only a few (like 3), he didn't really care one way or the other. I'm open to either, just want consistency!

Reply:
This was just missed after the rebase.  Changing.